woocommerce/woocommerce
# Exclude inactive plugins from the incompatible plugins view

### Changes proposed in this Pull Request:

Changes related to the warnings displayed in the plugins page for plugins that are incompatible with one or more features:

1. In the incompatible plugins page (`wp-admin/plugins.php?plugin_status=incompatible_with_feature`) include only the plugins that are currently active.
2. Show an empty page ("No plugins are currently available") when that same page is loaded without any incompatible plugin being active, instead of displaying all the available plugins.

### How to test the changes in this Pull Request:

1. Make sure that you have plugins that are incompatible with HPOS, some active and some inactive, and enable HPOS from the Settings - Advanced - Features page.
2. Click on "View all plugins" from the "You are viewing active plugins that are incompatible..." warning and verify that the plugins view that loads displays only the active plugins (and the counter in the view title is correct).
3. Reload the page after disabling all the conflicting plugins (you'll need to directly enter the URL as there's no link pointing to it anywhere now) and verify that the notice appears but the list of plugins is empty.

### Other information:




### FOR PR REVIEWER ONLY:
